setup/inject_SELinux_check/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+BEAKELIB_SCRIPT=/usr/share/beakerlib/logging.sh
```

Comment:
Typo here? beakeRlib_script (missing R)

Reply:
Yeah, that was typo, thanks for letting me know @sergio-correia